tests/bsim/bluetooth/host/l2cap/userdata/src/main_l2cap_userdata.c

```diff
@@ -14,8 +14,15 @@ extern enum bst_result_t bst_result;
 static struct bt_conn *default_conn;
 
 #define PSM 0x80
+#define DATA_SIZE 500
```

Comment:
this is to make sure we have many K-Frames (PDUs) to send an SDU. Since the bug was discovered with K-frames in the middle of an SDU.